File: Programmers/Inho/C#/Level01/3진법 뒤집기.cs
using System;
using System.Collections.Generic;
public class Solution
{
    public int solution(int n)
    {
        int answer = 0;
        List<int> three = new List<int>();
        while (n > 0)
        {
            three.Add(n % 3);
            n /= 3;
        }
        int[] temp = new int[three.Count];
        for (int i = 0; i < three.Count; i++)
        {
            temp[i] = three[i];
        }
        for (int i = 0; i < temp.Length; i++)
        {
            answer += temp[temp.Length - 1 - i] * (int)Math.Pow(3, i);
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/3진법 뒤집기.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>
#include <cmath>

using namespace std;

int solution(int n) {
    int answer = 0;
    string three = "";
    while(n/3!=0){
        three+=to_string(n%3);
        n/=3;        
    }
    three+=to_string(n%3);
    reverse(three.begin(), three.end());
    for(int i = 0; i<three.size();i++){
        answer+=(three[i]-'0')*pow(3, i);
    }
    return answer;
}